Dockerfile

```diff
@@ -90,7 +93,7 @@ RUN cd $ROOTFS/lib/modules && \
 RUN curl -L http://http.debian.net/debian/pool/main/libc/libcap2/libcap2_2.22.orig.tar.gz | tar -C / -xz && \
     cd /libcap-2.22 && \
     sed -i 's/LIBATTR := yes/LIBATTR := no/' Make.Rules && \
-    sed -i 's/\(^CFLAGS := .*\)/\1 -m32/' Make.Rules && \
+    sed -i 's/\(^CFLAGS := .*\)/\1 -m64/' Make.Rules && \
```

Comment:
We can just remove this `sed` completely at this point, right? (since `-m64` will be the proper default now)